src/main/kotlin/lotto/view/ConsoleResultView.kt

```diff
@@ -0,0 +1,48 @@
+package lotto.view
+
+import lotto.model.Lottos
+import lotto.model.WinningRank
+import lotto.model.WinningStatistics
+
+object ConsoleResultView : ResultView {
+
+    private const val principalRatio = 1
+
+    override fun printLottoCount(customLottoCount: Int, randomLottoCount: Int) {
+        println()
+        println("수동으로 ${customLottoCount}장, 자동으로 ${randomLottoCount}개를 구매했습니다.")
+    }
+
+    override fun printTotalLottos(lottos: Lottos) = lottos.lottos.forEach { println(it) }
+
+    override fun printWinningStatistics(paymentPrice: Int, statistics: WinningStatistics) {
+        println()
+        println("당첨 통계")
+        println("---------")
+
+        printWinningStatus(statistics)
+        printEarningsRatio(statistics.calculateEarningRatio(paymentPrice))
+    }
+
+    private fun printWinningStatus(statistics: WinningStatistics) =
+        statistics.winningStatistics.forEach { (winningRank, count) ->
+            println("${winningRank.matchedNumberCount}개 일치${bonusBallMessage(winningRank)} (${winningRank.prizeMoney}원)- ${count}개")
+        }
```

Comment:
코틀린에 with 이라는 녀석이 존재하는데요 with에 리시버로 받은 객체를 사용하여 아래와 같이 표현하는 방법도 있습니다~!
https://kotlinlang.org/docs/scope-functions.html#with

```suggestion
    private fun printWinningStatus(statistics: WinningStatistics) = with(StringBuilder()){
        statistics.winningStatistics.forEach { (winningRank, count) ->
            this.appendLine("${winningRank.matchedNumberCount}개 일치${bonusBallMessage(winningRank)} (${winningRank.prizeMoney}원)- ${count}개")
        }
        println(this)
    }
```